tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -933,9 +941,11 @@ def _InputFn():
     with self.assertRaisesRegex(
         errors.AbortedError,
         r"User disallowed engine native segment execution"):
-      converter.build(input_fn=_InputFn)
+      try:
+        converter.build(input_fn=_InputFn)
+      finally:
```

Comment:
Can you put this comment to the final block:
# Always reset the environment variable.

Reply:
Fixed